diff --git a/node_modules/@upstash/redis/dist/index.js b/node_modules/@upstash/redis/dist/index.js
index 3bbaaf4..6b31913 100644
--- a/node_modules/@upstash/redis/dist/index.js
+++ b/node_modules/@upstash/redis/dist/index.js
@@ -43,7 +43,7 @@ __export(pkg_exports, {
 });
 
 // pkg/http.ts
-var import_isomorphic_fetch = require("isomorphic-fetch");
+// var import_isomorphic_fetch = require("isomorphic-fetch");
 
 // pkg/error.ts
 var UpstashError = class extends Error {
diff --git a/node_modules/@upstash/redis/dist/index.mjs b/node_modules/@upstash/redis/dist/index.mjs
index a10f549..d649b30 100644
--- a/node_modules/@upstash/redis/dist/index.mjs
+++ b/node_modules/@upstash/redis/dist/index.mjs
@@ -111,7 +111,7 @@ import {
 } from "./chunk-RYSRH3HC.mjs";
 
 // pkg/http.ts
-import "isomorphic-fetch";
+// import "isomorphic-fetch";
 var HttpClient = class {
   constructor(config) {
     var _a;
